week-1/Homework/mandatory/2-logic-error.js

```diff
@@ -1,31 +1,40 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word;
 }
```

Comment:
In this function, you have been asked to use one of the string methods. That method deletes the white space between words. Google it and use it.